paymentsheet/src/main/java/com/stripe/android/common/model/CommonConfiguration.kt

```diff
@@ -25,6 +25,11 @@ internal data class CommonConfiguration(
     val externalPaymentMethods: List<String>,
     val cardBrandAcceptance: PaymentSheet.CardBrandAcceptance,
 ) : Parcelable {
+    private val isEKClientSecretValidRegexPattern = "^ek_[^_](.)+$"
```

Comment:
Looks like this is causing a compilation error because it's a member of this Parcelable class, but is not actually parcelable. I'd recommend moving it just out of the class definition (it could be a const val defined statically instead)

Reply:
:+1: